roles-and-groups.md

```diff
@@ -0,0 +1,153 @@
+- The voting process for removing a candiate is the same as the process for promoting a candidate (via PR).
+
+## Definition of groups
+
+Since Hiero contains multiple projects, we define GitHub **groups** that are used to manage the access to the GitHub repositories.
+The groups are defined in the `config.yaml` file and are used to manage the access to the GitHub repositories that belong to a project.
+Contributors do not have any specific rights, and therefore we do not need to define any groups for them.
+Committers and Maintainers are defined in groups that are used to manage the access to the GitHub repositories.
+For each project we have defined a Committers group and a Maintainers group:
+
+- **`PROJECT-maintainers`**: For each project we have a group that defines the Maintainers of that project.
+- **`PROJECT-committers`**: For each project we have a group that defines the Committers of that project.
+
+There are general groups in the [`config.yaml`](https://github.com/hiero-ledger/governance/blob/main/config.yaml) file as well:
+
+- **`hiero-automation`**: This group contains the CI/CD Automation service accounts for hiero-ledger projects.
+- **`github-maintainers`**: This group contains those responsible for maintaining our GitHub environment and infrastructure (including GitHub Action runners, ...).
+- **`tsc`**: This group contains all members of the technical steering committee (TSC) as defined [here](https://github.com/hiero-ledger/tsc).
+- **`lf-staff`**: This group contains all Linux Foundation and LF Decentralized Trust members who require (mostly ADMIN) rights on the organization.
+- **`security-maintainers`**: This group contains members of the security teams who are responsible for monitoring and maintaining security tooling and automated scanners within hiero-ledger projects.
+- **`prod-security`**: This group contains members of the security teams who are responsible for reviewing code for security issues, threat modeling and reducing risk within hiero-ledger projects.
+- **`sec-ops`**: This group contains members of the security teams who are responsible for responding to security incidents within hiero-ledger projects.
+
+## Definitions of roles
+
+For each project the following GitHub roles should be defined for the groups that work on the project.
+The GitHub roles define the privileges afforded the groups on the GitHub repositories of the project.
+All roles are based on the [GitHub role defintions](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization).
+
+- **`github-maintainers`**: `MAINTAIN`
+- **`tsc`**: `MAINTAIN`
+- **`PROJECT-maintainers`**: `MAINTAIN`
+- **`PROJECT-committers`**: `WRITE`
+
+The GitHub roles are assigned in the [`config.yaml`](https://github.com/hiero-ledger/governance/blob/main/config.yaml).
```

Comment:
Are changes to Github roles or groups also changed via the nomination process? (Obviously there needs to be a PR, but is voting required?)

Reply:
Good point. We should not define the full processes for voting but ignore how the process to add "security-maintainers" to a repo looks like. Who does that or vote for that?